app/Http/Controllers/Admin/RecommendedPlanController.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Area;
+use App\Models\City;
+use App\Models\Prefecture;
+use Illuminate\Http\Request;
+use App\Models\Genre;
+use App\Models\Interest;
+use App\Models\Place;
+use App\Models\User;
+use App\Models\Keyword;
+use App\Models\PlaceImage;
+use App\Models\PlaceKeyword;
+use App\Models\Plan;
+use App\Models\PlanDetail;
+use App\Models\PlanFavorite;
+use App\Models\PlanKeyword;
+use Illuminate\Pagination\Factory;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\View;
+use Illuminate\Pagination\Paginator;
+
+class RecommendedPlanController extends Controller
+
+{
+    private $recommended_plan;
+
+    public function __construct(Plan $recommended_plan){
+        $this->recommended_plan = $recommended_plan;
+    }
+
+    //----- For RECOMMENDED PLANS LIST PAGE -----
+
+    # Use this to show the list of the Recommended plans
+    public function index(){
+        $recommended_plans = Plan::where('user_type', '=' , 'admin')->paginate(10);
+
+        return view('admin.plans.recommended_plans', [        
+            'recommended_plans' => $recommended_plans, 
+        ]); 
+    } 
+        
+    public function store(Request $request){
+            $request->validate
+            ([
+                'title' => 'required|max:15|unique:plans,title',
+            ]);
+            
+            $recommended_plan = new Plan;
+            $recommended_plan->title =  ucwords(strtolower($request->title));
+            $recommended_plan->user_id = Auth::user()->id;
+            $recommended_plan->user_type = 'admin';
+            $recommended_plan->save();
+
+            return redirect()->route('recommended_plans');
+    }
+
+    public function update($id, Request $request){
```

Comment:
you can do the same implementation with the `destroy` function below when you inject the `Plan` model instead of `$id`